egs/wsj/s5/steps/nnet3/tdnn/train.sh

```diff
@@ -5,7 +5,7 @@
 # Copyright 2012-2015  Johns Hopkins University (Author: Daniel Povey).
 #           2013  Xiaohui Zhang
 #           2013  Guoguo Chen
-#           2014  Vimal Manohar
+#           2014-2016  Vimal Manohar
```

Comment:
it would be great if you could move this code to steps/nnet3/train_dnn.py. This will help us use the same script for a variety of architectures.